common/src/main/java/edu/snu/nemo/common/ir/vertex/MetricCollectionBarrierVertex.java

```diff
@@ -29,7 +29,9 @@
  */
 public final class MetricCollectionBarrierVertex<T> extends IRVertex {
   // Partition ID to Size data
```

Comment:
Block ID to size data?

Reply:
BTW, why don't we change it to task ID? Block ID seems quite tied to our current data skew handling example.

Reply:
It would have been great only if we can extract block id from task id. But unfortunately, currently block id cannot be extracted from task id and vice versa.

Block id requires runtime edge id which cannot be parsed from task id.
Task id requires stage id which cannot be parsed from block id.

I think this problem will be naturally handled with NEMO-100, via refactoring RuntimeIdGenerator.